admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+----
+
+Once the builds have successfully completed, the image streams will be populated
+and ready for deployment. However, you must first specify the `*IDEX_PREFIX*`
+parameter with the registry location.
+
+. Run the deployer, specifying the host on which kibana will run, and the
+master URL where browsers will be directed to when authenticating to OpenShift:
++
+====
+----
+oc process logging-deployer-template \
+           -v KIBANA_HOSTNAME=kibana.example.com,PUBLIC_MASTER_URL=https://localhost:8443 \
+           | oc create -f -
+----
+====
++
+You can use the following environment variables when processing the kibana
+service:
++
+[cols="2",options="header"]
+|===
+|Variable Name
+|Description
+
+|`*KIBANA_HOSTNAME*`
+|(Required with the 'oc process' command) The external host name for web clients
+to reach Kibana.
+
+|`*PUBLIC_MASTER_URL*`
+|(Required with the 'oc process' command) The external URL for the master. For
+OAuth use.
+
+|`*IMAGE_PREFIX*`
+|The prefix for logging component images. For example, setting the prefix to
+"openshift/origin-" creates "openshift/origin-logging-deployer:v1.1".
+
+|`*IMAGE_VERSION*`
+|The version for logging component images. For example, setting the version to
+"v1.1" creates "openshift/origin-logging-deployer:v1.1".
+
+|`*ES_INSTANCE_RAM*`
+|Amount of RAM to reserve per ElasticSearch instance. The default is 8GB, and it must be at least 512MB.
+
+|`*ES_CLUSTER_SIZE*`
+|The amount of instances of ElasticSearch to deploy. Redundancy requires at
+least three, and more can be used for scaling.
+
+|`*ENABLE_OPS_CLUSTER*`
+|If set to "true", configures a second ElasticSearch cluster and Kibana for
+operations logs.
+
+|`*KIBANA_OPS_HOSTNAME*`, `*ES_OPS_INSTANCE_RAM*`, `*ES_OPS_CLUSTER_SIZE*`
+|Variables for the operations log cluster.
+|===
++
+When setting the `*ENABLE_OPS_CLUSTER*` parameter to 'true', Fluentd splits logs
+between the ElasticSearch cluster and a cluster reserved for operations logs.
+This means a second ElasticSearch and Kibana are deployed. The deployments are
+distinguishable by the '-op' included in their names.
+
+. Next, as a cluster administrator, deploy the templates that were created by
+the deployer:
++
+====
+----
+$ oc process logging-support-template | oc create -f -
+----
+====
+
+. Enable the Fluentd service account by editing the security context:
++
+====
+----
+$ oc edit scc/privileged
+----
+====
++
+And add the following user to the file:
++
+====
+----
+- system:serviceaccount:logging:aggregated-logging-fluentd
+----
+====
+
+. Give the service account permission to read labels from all pods:
++
+====
+----
+$ openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
+----
+====
+
+== Post-deployment Configuration
+
+=== ElasticSearch
+
+All pods in OpenShift deployments can safely share any volumes specified in the
+deployment. With ElasticSearch, all pods must have their own storage.
```

Comment:
It's not very clear here that ElasticSearch usage is a contrast to the standard use case. And it needs to clear that the first sentence describes the standard use case, not an assurance of safety. It's a head-scratcher. Maybe:

"All pods from an OpenShift deployment share any volumes specified for the deployment. However, ElasticSearch pods cannot share storage."